build/ci-pipeline-mag.yml

```diff
@@ -0,0 +1,86 @@
+# DESCRIPTION: 	
+# Tests deployment to Microsoft Azure Government
+
+name: $(SourceBranchName)-$(Date:yyyyMMdd)$(Rev:-r)
+trigger: none
+
+variables:
+  ConnectedServiceName: 'mshapis-govtest-deploymentagent'
+  ResourceGroupRegion: 'usgovvirginia'
+  AppInsightsRegion: 'usgovvirginia'
+  SqlRegion: 'usgoviowa'
+  resourceGroupRoot: 'msh-fhir-ci'
+  appServicePlanResourceGroup: '$(resourceGroupRoot)'
+  appServicePlanName: '$(resourceGroupRoot)-asp'
+  DeploymentEnvironmentName: '$(resourceGroupRoot)'
+  DeploymentEnvironmentNameSql: '$(DeploymentEnvironmentName)-sql'
```

Comment:
Only the R4 and R4Sql deployment environment names and test environments are needed as the other versions aren't used.